Sources/Vapor/Application.swift

```diff
@@ -94,6 +94,10 @@ public final class Application {
         DotEnvFile.load(for: environment, on: .shared(self.eventLoopGroup), fileio: self.fileio, logger: self.logger)
     }
     
+    /// Starts the Application using the `start()` command, then waits for any running tasks to complete
+    /// If your application is started without arguments, the default argument is used.
+    ///
+    /// Under normal circumstances, `run `will start the shut down and wait for the web server to (manually) shut down before exiting the application.
```

Comment:
```suggestion
    /// Under normal circumstances, `run() `will begin the shutdown, then wait for the web server to (manually) shut down before returning.
```
(Yes, the use of both "shut down" and "shutdown" is on purpose 😅)